### PR TITLE
add table to UG and DG

### PR DESCRIPTION
**Pull request by brian16600**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -581,6 +582,27 @@ testers are expected to do more *exploratory* testing.
...
+|-----------|---------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `n/`      | Name of contact | Can only contain alphanumeric characters and selected symbols like `/`, `-` and `'`.                                                                                                                                                                                                                                                                                                                                                                                                       |
```
Comment (snss231) [inline docs/DeveloperGuide.md:589]: And spaces

Comment (snss231) [reply docs/DeveloperGuide.md:589]: No numbers (only alphabets)

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -581,6 +582,27 @@ testers are expected to do more *exploratory* testing.
...
+| `p/`      | Phone number of contact | Only numbers are allowed and must contain at least 3 digits. Only used in `addc` and `editc` commands.                                                                                                                                                                                                                                                                                                                                                                                     |
```
Comment (snss231) [inline docs/DeveloperGuide.md:590]: May be confusing that there are two separate entries for `p/`... maybe best to combine the two into a row and explain in the cells

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -581,6 +582,27 @@ testers are expected to do more *exploratory* testing.
 
 2. _{ more test cases …​ }_
 
+## **Prefix Summary**
+
+| Prefix    | Meaning | Constraints                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```
Comment (snss231) [inline docs/DeveloperGuide.md:587]: May be good to have a column to list where the commands are used

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -581,6 +582,27 @@ testers are expected to do more *exploratory* testing.
...
+| `tn/`     | Task name of the task | Two tasks with the same `TASKNAME` is valid.                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| `dt/`     | Date and time of the task. Can include both start and end times, or simply a single time. | Needs to be in the format dd-mm-yyyy hhmm                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```
Comment (snss231) [inline docs/DeveloperGuide.md:597]: `dt/` does not always allow a single date time (e.g. in findt)

**Review by snss231 — COMMENTED** [review]
Left some feedback regarding the table

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -581,6 +582,27 @@ testers are expected to do more *exploratory* testing.
 
 2. _{ more test cases …​ }_
```
Comment (snss231) [inline docs/DeveloperGuide.md:583]: btw lets remove this

**Comment by brian16600** [comment]
Fixed, going ahead to merge

**Pull request merged**